src/app/theme/themeManager.ts

```diff
@@ -41,23 +43,22 @@ export function setHomeTheme(params: HomeThemeParams): void {
       flagshipUIEvents.SET_COMPONENT_COLORS,
       componentId,
       {
-        topTheme: 'light',
-        bottomTheme: 'light'
+        topTheme: Appearance.getColorScheme() === 'dark' ? 'dark' : 'light',
+        bottomTheme: Appearance.getColorScheme() === 'dark' ? 'dark' : 'light'
       }
     )
   } else {
     flagshipUIEventHandler.emit(
       flagshipUIEvents.SET_COMPONENT_COLORS,
       componentId,
       {
-        topTheme: 'dark',
-        bottomTheme: 'dark'
+        topTheme: Appearance.getColorScheme() === 'dark' ? 'light' : 'dark',
+        bottomTheme: Appearance.getColorScheme() === 'dark' ? 'light' : 'dark'
       }
     )
   }
 
   homeThemeRawValue = homeTheme
-  themeLog.info(`setHomeTheme: ${homeTheme}`)
```

Comment:
is that expected that we removed this log?

Reply:
Not at all I fix it